src/natcap/invest/ui_server.py

```diff
@@ -202,39 +193,9 @@ def save_to_python():
     payload = request.get_json()
     save_filepath = payload['filepath']
     modelname = payload['modelname']
-    pyname = payload['pyname']
     args_dict = json.loads(payload['args'])
 
-    script_template = textwrap.dedent("""\
-    # coding=UTF-8
-    # -----------------------------------------------
-    # Generated by InVEST {invest_version} on {today}
-    # Model: {modelname}
-
-    import {py_model}
-
-    args = {model_args}
-
-    if __name__ == '__main__':
-        {py_model}.execute(args)
-    """)
-
-    with codecs.open(save_filepath, 'w', encoding='utf-8') as py_file:
-        # cast_args = dict((unicode(key), value) for (key, value)
-        #                  in args_dict.items())
-        args = pprint.pformat(args_dict, indent=4)  # 4 spaces
-
-        # Tweak formatting from pprint:
-        # * Bump parameter inline with starting { to next line
-        # * add trailing comma to last item item pair
-        # * add extra space to spacing before first item
-        args = args.replace('{', '{\n ')
-        args = args.replace('}', ',\n}')
-        py_file.write(script_template.format(
-            invest_version=natcap.invest.cli.__version__,
-            today=datetime.now().strftime('%c'),
-            modelname=modelname,
-            py_model=pyname,
-            model_args=args))
+    cli.export_to_python(
```

Comment:
I think we always intended to make use of this `cli` function here, but must have forgotten to make the switch earlier.